app/models/policy.rb

```diff
@@ -63,6 +65,8 @@ module Actions
     DELETE_ROLE = "role:delete"
     ASSIGN_USER_ROLE = "role:assignUser"
     REMOVE_USER_ROLE = "role:removeUser"
+
+    UPDATE_USER = 'user:edit'
```

Comment:
Can we change this to "user:update"? Just for consistency with all the other actions.